### PR TITLE
032 update readme

### PR DESCRIPTION
**Pull request by conormcgee480**
https://trello.com/c/G1MHGi5q

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,34 @@
...
+    $ npm install
+
+Once all of the dependencies have finished installing, you can start the application by running the following command.
+
+    $ npm start
+
+### Testing
+You can run the tests contained in the project's test folder simply by running the following command.
+
+    $ npm test
```
Comment (seaninbreslin) [inline README.md:31]: npm test will only run the UI tests. The command I gave you will run all the tests including the automation ones

**Comment by conormcgee480** [comment]
[US032 - Update README for running and testing (1/2)](https://trello.com/c/G1MHGi5q/28-us032-update-readme-for-running-and-testing-1-2)

**Review by seaninbreslin — APPROVED** [review]
Asides from the one comment I made about test commands - looks all good to me 😄

File: README.md (hunk 1 above)
Comment (conormcgee480) [reply README.md:31]: Ah no worries, will edit that now.

**Comment by conormcgee480** [comment]
[US032 - Update README for running and testing (1/2)](https://trello.com/c/G1MHGi5q/28-us032-update-readme-for-running-and-testing-1-2)

**Review by seaninbreslin — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1 +1,37 @@
...
+    
+    $ npm test
+
+You can run the application's automation tests by running the following command.
+
+    $ npm test ./test/JobRolesAutomationTest.js
+
+
+    
+
```
Comment (boydstepnew) [inline README.md]: few extra lines, let's remove those

**Review by boydstepnew — APPROVED** [review]

**Pull request merged**